### ISSUE TITLE
[Support]: Cache breaks on every reboot

### ISSUE THREAD

**Issue by guniv** [blakeblackshear/frigate#3017]
### Describe the problem you are having

I recently moved my setup to a new house and was in the process of setting up a new camera in Frigate. The initial addition of the camera worked well, and I used the mask tool to make a mask and then rebooted Frigate to apply the mask. On reboot, I started receiving these errors: 

```
frigate.record ERROR : Error occurred when attempting to maintain recording cache

frigate.record ERROR : time data '%Y%m%d%H%M%S' does not match format '%Y%m%d%H%M%S'

```
I am running the latest version of Frigate via UNRAID. I tried a bunch of things to solve this:

Changed the location of Frigate folder in appdata
Reverted to a previous version of Frigate (I believe 0.9.4)?

I FINALLY got it to work again by changing the cache location during the setup process for Frigate in UNRAID. I changed it from tmp/frigate to tmp/frigate2 and it worked, the camera showed up looking good, and I received no errors. I was skeptical, though, and initiated a reboot of Frigate from the Frigate UI (which actually only shut it down). I booted it back up and... same errors began looping over and over again.

I'm pretty stumped at this point with my limited knowledge.

### Version

10.1

### Frigate config file

```yaml
mqtt:
  host: [REMOVED]
  user: [REMOVED]
  password: [REMOVED]
ffmpeg:
  hwaccel_args: 
    - -hwaccel
    - vaapi
    - -hwaccel_device
    - /dev/dri/renderD128
    - -hwaccel_output_format
    - yuv420p
cameras:
  front_door:
    ffmpeg:
      output_args:
        record: -c copy -c:a aac
        rtmp: -c copy -f flv -c:a aac
      inputs:
        - path: rtsp://[REMOVED]@192.168.1[REMOVED]:554/cam/realmonitor?channel=1&subtype=0&authbasic=64
          roles:
            - detect
            - rtmp
            - record
    objects:
      track:
       - person
    record:
      enabled: true
      events:
        retain:
          default: 14
    detect:
      width: 2560
      height: 1920
      fps: 5
    snapshots:
      # Optional: Enable writing jpg snapshot to /media/frigate/clips (default: shown below)
      # This value can be set via MQTT and will be updated in startup based on retained value
      enabled: true
      # Optional: print a timestamp on the snapshots (default: shown below)
      timestamp: true
      # Optional: draw bounding box on the snapshots (default: shown below)
      bounding_box: true
      # Optional: crop the snapshot (default: shown below)
      crop: false
      # Optional: height to resize the snapshot to (default: original size)
      height: 300
      # Optional: Camera override for retention settings (default: global values)
      retain:
        # Required: Default retention days (default: shown below)
        default: 10
    rtmp:
      # Required: Enable the live stream (default: True)
      enabled: True
    zones:
      not_road:
        coordinates: 2560,1920,2514,1189,1565,1320,727,1276,556,1296,526,403,0,400,0,1358,0,1920
        objects:
          - person

detectors:
  coral:
    type: edgetpu
    device: usb
```

### Relevant log output

```shell
[2022-03-25 21:45:19] frigate.record ERROR : Error occurred when attempting to maintain recording cache

[2022-03-25 21:45:19] frigate.record ERROR : time data '%Y%m%d%H%M%S' does not match format '%Y%m%d%H%M%S'

[2022-03-25 21:45:24] watchdog.front_door ERROR : Ffmpeg process crashed unexpectedly for front_door.

[2022-03-25 21:45:24] watchdog.front_door ERROR : The following ffmpeg logs include the last 100 lines prior to exit.

[2022-03-25 21:45:24] ffmpeg.front_door.detect ERROR : Multiple -c, -codec, -acodec, -vcodec, -scodec or -dcodec options specified for stream 1, only the last option '-c:a aac' will be used.

[2022-03-25 21:45:24] ffmpeg.front_door.detect ERROR : File '/tmp/cache/front_door-%Y%m%d%H%M%S.mp4' already exists. Overwrite? [y/N] Not overwriting - exiting
```

### FFprobe output from your camera

```shell
can pull if needed
```

### Operating system

UNRAID

### Install method

Docker CLI

### Coral version

USB

### Network connection

Wired

### Camera make and model

Amcrest AD410

**Comment by blakeblackshear** [comment]
Strange. This means some file is getting created in the cache directory with an unexpected file name. Can you list the contents of that directory?

**Comment by blakeblackshear** [comment]
I see. You are overriding the output args with a set that are invalid. You can't completely replace the output args. You need to copy the default args and adjust them from there.

**Comment by blakeblackshear** [comment]
You want

```yaml
record: -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c:v copy -c:a aac 
```

**Comment by guniv** [comment]
Thanks! I had copied the arguments from another camera I was previously using (a Wyze Cam V3) and mindlessly cleared those arguments since I thought they only applied to that camera. The AD410 now shows up again on reboots. However, the errors on maintaining recording cache do continue to occur and it won't save recordings. It IS fixed, however, if I set a new cache folder (everything continues to work on a reboot, unlike before).

My assumption is that I somehow need to get into the original cache folder and delete old files but I frankly have no idea how to get into that folder in UNRAID. But for now, just making a new cache folder to circumvent the issue works fine for me. Thank you for your help and an amazing project!
